source/mainnet/smart-contracts/guides/invoke-instance.rst

```diff
@@ -39,13 +39,12 @@ If successful, the output should be similar to the following:
 
 .. code-block:: console
 
-   Successfully updated contract instance {"index":0,"subindex":0} using the entrypoint 'my_receive'.
    Invocation resulted in success:
    - Energy used: 595 NRG
    - Return value:
        <some-return-value>
 
-As you can see, the subindex defaults to ``0``.
+As you can see, the subindex defaults to ``0``. If `my_receive` updates the contract state, these changes are not persisted to the chain and are discarded.
```

Comment:
Not related to the PR's issue, but

> As you can see, the subindex defaults to ``0``

feels a bit confusing. I don't think I can see from the example how the subindex defaults to ``0``.  Maybe we could just say before we give the example that subindex defaults to zero.